### PR TITLE
WT-8709 Do not hold the oldest id of logged tables

### PR DESCRIPTION
**Pull request by raviprakashgiri29**

### CODE REVIEW

File: src/include/txn_inline.h [hunk 1]
```diff
@@ -476,7 +476,8 @@ __wt_txn_oldest_id(WT_SESSION_IMPL *session)
      */
     WT_READ_BARRIER();
 
-    if (!F_ISSET(conn, WT_CONN_RECOVERING)) {
```
Comment (kommiharibabu) [inline src/include/txn_inline.h:479]: Can you add a comment explaining the reason why the logged tables are not required to consider the recovered checkpoint snapshot as a minimum? The logged tables are not rollbacked as part of RTS, so there is no point in holding their oldest_id.

**Review by quchenhao — COMMENTED** [review]
Can you change the title of the pr? It doesn't tell much about the solution.

File: src/include/txn_inline.h (hunk 1 above)
Comment (raviprakashgiri29) [reply src/include/txn_inline.h:479]: Done

**Comment by raviprakashgiri29** [comment]
> Can you change the title of the pr? It doesn't tell much about the solution.

Done

**Review by quchenhao — APPROVED** [review]
lgtm

File: src/include/txn_inline.h (hunk 1 above)
Comment (kommiharibabu) [reply src/include/txn_inline.h:479]: `rollbacked` is not a proper word I think. How about something like the following?
`The logged tables are excluded as part of RTS, so there is no need of holding their oldest_id`?

Comment (raviprakashgiri29) [reply src/include/txn_inline.h:479]: This makes more sense. Thanks.

**Review by kommiharibabu — APPROVED** [review]
LGTM

**Pull request merged**